NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/blockinteract/BlockInteractListener.java

```diff
@@ -190,7 +190,8 @@ else if (MovingUtil.hasScheduledPlayerSetBack(player)) {
         if (block == null) {
             data.resetLastBlock();
             blockChecks = false;
-        } else if (block.getType().toString().equals("SCAFFOLDING")) {
+        } 
+        else if (BlockProperties.isScaffolding(block.getType())) { // null check included in BlockProperties.
         	blockChecks = false;
         }

```

Comment:
I think this should be removed, for now. Let it record to LastBlock below instead of returning null on bIData#getLastType().